editors/code/src/config.ts

```diff
@@ -172,6 +172,15 @@ export class Config {
         };
     }
 
+    get highlightRelated() {
+        return {
+            references: this.get<boolean>("highlightRelated.references"),
+            exitPoints: this.get<boolean>("highlightRelated.exit_points"),
+            breakPoints: this.get<boolean>("highlightRelated.exit_points"),
+            yieldPoints: this.get<boolean>("highlightRelated.yield_points")
+        };
+    }
```

Comment:
I think this is dead code and should be removed?

Looks perfect to me otherwise, thanks!

bors d+

Reply:
Makes sense to me to remove, I was just mimicking other configuration here.

Reply:
other stuff is used on the VS Code side as well I belive. Although if someth things are unsued, might makes sense to remove thouse